scenarios/langchain/tracing-with-langchain.ipynb

```diff
@@ -0,0 +1,312 @@
+    "\n",
+    "4. Go to the section **Tracing** in Azure AI Foundry portal and copy the instrumentation connection string.\n",
+    "5. Configure the following environment variables (or update the code in the notebook directly):\n",
+    "\n",
+    "    * `AZURE_APPINSIGHT_CONNECTION`: The instrumentation connection string, pointing to the Azure Application Insights used by the hub or project.\n",
+    "    * `AZURE_INFERENCE_ENDPOINT`: The endpoint where the models are deployed.\n",
+    "    * `AZURE_INFERENCE_CREDENTIAL`: The credentials to connect to the mode."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Install the following packages:\n",
+    "\n",
+    "```bash\n",
+    "pip install -U langchain-core langchain-azure-ai[opentelemetry]\n",
+    "```\n",
+    "\n",
+    "> Notice that we are installing the extra `langchain-azure-ai[opentelemetry]` which allows instrumentation via opentelemetry with LangChain in Azure AI Foundry."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Get the connection string to Application Insights"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can use the tracing capabilities in Azure AI Foundry by creating a tracer. Logs are stored in Azure Application Insights and can be queried at any time and hence you need a connection string to it. Each AI Hub has an Azure Application Insights created for you. You can get the connection string by **either**:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using the connection string directly:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "application_insights_connection_string = os.environ[\"AZURE_APPINSIGHT_CONNECTION\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using the Azure AI Foundry SDK\n",
+    "\n",
+    "You can also get the connection string to Application Insights by using the Azure AI Foundry SDK along with the connection string to the project, as follows:\n",
+    "\n",
+    "Install the Azure AI Foundry SDK:\n",
+    "\n",
+    "```bash\n",
+    "pip install azure-ai-projects\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.projects import AIProjectClient\n",
+    "from azure.identity import DefaultAzureCredential\n",
+    "\n",
+    "project_client = AIProjectClient.from_connection_string(\n",
+    "    credential=DefaultAzureCredential(),\n",
+    "    conn_str=\"<region>.api.azureml.ms;<tenant>;<hub>;<project>\",\n",
```

Comment:
We shouldn't document the format of the connection string, it implies that they need to construct it manually. Suggest to:
 - change this to <project_connection_string>
 - tell them where to get the connection string from (the project overview page)

Reply:
Done!